docs/topics/wasm/wasm-overview.md

```diff
@@ -1,61 +1,75 @@
+Kotlin has the power to build applications, and reuse mobile and desktop user interfaces (UIs) in your web projects with
+Compose Multiplatform for Web and Kotlin/Wasm.
 
-Kotlin/Wasm is a new compilation target for Kotlin. You can use it in your Kotlin Multiplatform projects.
-With Kotlin/Wasm, you can create applications that run on different environments and devices supporting WebAssembly and meeting Kotlin's requirements.
+[Compose Multiplatform]((https://www.jetbrains.com/lp/compose-multiplatform/)) is a declarative framework based on Kotlin
+and [Jetpack Compose](https://developer.android.com/jetpack/compose), which allows you to implement the UI
+once and share it across all the platforms you target. Specifically for web platforms, Compose Multiplatform for Web uses 
+Kotlin/Wasm as its compilation target.
 
-> Learn more about Kotlin/Wasm with this [YouTube playlist](https://kotl.in/wasm-pl).
+Explore our online demo of an application built with Compose Multiplatform for Web and Kotlin/Wasm:
+
+<a href="https://zal.im/wasm/jetsnack/"><img src="wasm-demo.png" width="700" alt="Kotlin/Wasm demo"/></a>
+
+>To run applications built with Kotlin/Wasm in a browser, you need a browser version that supports the new garbage collection
+> and exception handling proposals. To check the support status, see the [WebAssembly
+> roadmap](https://webassembly.org/roadmap/).
 >
-{type="note"}
+{type="tip"}
 
-## Browser support
+[WebAssembly (Wasm)](https://webassembly.org/) is a binary instruction format for a stack-based virtual machine. This
+format is platform-independent because it runs on its own virtual machine. Wasm provides languages like Kotlin, and many 
+more, with a compilation target so that they can run on the web.
 
-To run applications built with Kotlin/Wasm in a browser, you need a version supporting the new [garbage collection feature](https://github.com/WebAssembly/gc).
+Kotlin/Wasm compiles your Kotlin code into Wasm format. Using Kotlin/Wasm, you can create applications that run on 
+different environments and devices, which support Wasm and meet Kotlin's requirements.
 
-[Learn more in Get started with Kotlin/Wasm](wasm-get-started.md#troubleshooting).
+Would you like to try it yourself?
 
-## Interoperability
+<a href="wasm-get-started.md"><img src="wasm-get-started-button.png" width="700" alt="Get started with Kotlin/Wasm"/></a>
 
-Kotlin/Wasm allows you to both use JavaScript code and Browser API from Kotlin, and Kotlin code from JavaScript.
+## Performance
 
-[Learn more about Kotlin Wasm interoperability with JavaScript](wasm-js-interop.md).
+Although Kotlin/Wasm is still in Alpha, Compose Multiplatform running on Kotlin/Wasm already shows encouraging performance 
+traits. You can see that the execution speed outperforms JS and is getting close to JVM:
 
-## Compose Multiplatform for Web
+![Kotlin/Wasm performance](wasm-performance-compose.png){width=700}
```

Comment:
It's not clear what is on the Y axis... Can we ask to update the chart and add the data/legend?

Reply:
Checking with SME to find who created it and see if we can update it.

Reply:
Changed (: